pkgs/development/compilers/nim/default.nix

```diff
@@ -30,10 +29,16 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     makeWrapper nodejs-slim-8_x tzdata coreutils
```

Comment:
`makeWrapper tzdata coreutils` should be in `nativeBuildInputs` (they're needed only in installPhase and checkPhase)